src/robotFactory.js

```diff
@@ -1,10 +1,70 @@
 'use strict';
 
-class BaseRobot {}
+class BaseRobot {
+  constructor(name, weight, coords, chipVersion) {
+    this.name = name;
+    this.weight = weight;
+    this.coords = coords;
+    this.chipVersion = chipVersion;
+    this.coords.x = coords.x || 0;
+    this.coords.y = coords.y || 0;
+  }
 
-class FlyingRobot {}
+  goForward(step = 1) {
+    this.coords.y += step;
+  }
 
-class DeliveryDrone {}
+  goBack(step = 1) {
+    this.coords.y -= step;
+  }
+
+  goRight(step = 1) {
+    this.coords.x += step;
+  }
+
+  goLeft(step = 1) {
+    this.coords.x -= step;
+  }
+
+  getInfo() {
+    return `Robot: ${
+      this.name}, Chip version: ${
+      this.chipVersion}, Weight: ${this.weight}`;
+  }
```

Comment:
Don't forget about good code formatting. So that the linter does not quarrel, you can make our string a little bit into parts:
```suggestion
  getInfo() {
    return `Robot: ${this.name}, ` +
          `Chip version: ${this.chipVersion}, ` +
          `Weight: ${this.weight}`;
  }
```